prompts/git/README.md

```diff
@@ -0,0 +1,43 @@
+---
+extractors:
```

Comment:
As far as I can see, you're no longer using this as a prompt.  You're using a git-branches function but this is not a standalone prompt in your use case.